cmd/init.go

```diff
@@ -130,9 +130,10 @@ func init() {
 
 	initCmd.Flags().BoolVar(&install.WithoutCNI, "without-cni", false, "If true we not install cni plugin")
 	initCmd.Flags().StringVar(&install.Network, "network", "calico", "cni plugin, calico..")
+	initCmd.Flags().StringVar(&install.CniRepo, "cni-repo", "", "cni repo image, cilium..")
 	initCmd.Flags().BoolVar(&install.IPIP, "ipip", true, "ipip mode enable, calico..")
 	initCmd.Flags().StringVar(&install.MTU, "mtu", "1440", "mtu of the ipip mode , calico..")
-
+	initCmd.Flags().BoolVar(&install.SkipKubeProxy, "skip-kubeproxy", false, "skip install kube-proxy, only cilium")
```

Comment:
这个参数同理，直接检测cni类型如果是cilium就直接skip呗 我理解不需要多加这个参数

Reply:
@fanux cilium目前默认是共存的

Reply:
也有用cilium但是开启kube-proxy的

Reply:
@fanux  可能有两种情况, `cilium with/without  kube-proxy` .  官网推荐的是不安装 `kube-proxy.`  这个开关是给用户的,  默认跳过`kube-proxy`?

Reply:
我觉得少就是多，既然cilium推荐不装kube-proxy 咱就不装。 用cilium的人不要给那么多选择我认为，给个最好的选择就行。      除非能给出充分的用cilium还需要用kube-proxy的理由

Reply:
同意， without  kube-proxy

Reply:
Cilium 并非不能用于某些低版本内核，但是如果去掉kube-proxy就必须高版本内核了，这个条件估计还是有点高。现在的选项--SkipKubeProxy=True和Flase经过测试都是可以工作的，可以设置缺省为True

```html
Cilium’s kube-proxy replacement depends on the Host-Reachable Services feature, therefore a v4.19.57, v5.1.16, v5.2.0 or more recent Linux kernel is required. Linux kernels v5.3 and v5.8 add additional features that Cilium can use to further optimize the kube-proxy replacement implementation.

Note that v5.0.y kernels do not have the fix required to run the kube-proxy replacement since at this point in time the v5.0.y stable kernel is end-of-life (EOL) and not maintained anymore on kernel.org. For individual distribution maintained kernels, the situation could differ. Therefore, please check with your distribution.
```

Reply:
@fanux @ysicing 
为了支持市面上更多现存的内核版本，cilium缺省是和kube-proxy一起工作的，不带kube-proxy是新内核才能做到。现在既然两方面都已经工作得很好，建议不要去掉with kube-proxy选择。你觉得怎样？

Reply:
@divfor 我觉得用cilium的人群就不应该考虑在低版本内核上玩，中间过度方案支持没有意思，sealos需要向终态看齐，有的时候需要强势抛弃一些东西让工具更简单，几遍一小部分人用着别扭